src/api/auth/authorization.ts

```diff
@@ -53,7 +52,7 @@ async function getUserInfo(req: APIGatewayProxyEvent, config: Config): Promise<U
 
     user['is_superuser'] = user['cognito:groups'].includes('animl_superuser');
     user['projects'] = projects;
-    user['curr_project_roles'] = projects[selectedProject] ? projects[selectedProject].roles : [];
+    user['curr_project_roles'] = selectedProject ? projects[selectedProject].roles : [];
```

Comment:
I think this needs to be reverted, since projects might not have the selectedProject property

Reply:
@jue-henry we do need to support calls with undefined `selectedProject`, because in at least one call (the initial `getProjects` call when the app is loading) the user doesn't yet have any Projects let alone a selected one. 

I wrote up an issue to figure out a better way to handle that here: https://github.com/tnc-ca-geo/animl-api/issues/237

Reply:
I reverted it back to errors in this [commit](https://github.com/tnc-ca-geo/animl-api/pull/236/commits/be820d3a7b7da0c5a6a068bcc0a42e118e647282).

Reply:
@nathanielrindlaub Are you going to resolve the issue in this PR? Currently as superuser who does not have permissions in any other projects, this throws a 500 error.